src/Activity.js

```diff
@@ -0,0 +1,74 @@
+class Activity {
+  constructor(activityData) {
+    this.activityData = activityData;
+  }
+
+  getUserData(id) {
+    return this.activityData.filter(obj => obj.userID === id);
+  }
+
+  getDay(id, date) {
+    return this.getUserData(id).find(obj => obj.date === date);
+  }
+
+  getWeek(id, date) {
+    let endDate = this.getUserData(id).findIndex(obj => obj.date === date);
+    return endDate - 6 >= 0 ?
+      this.getUserData(id).slice(endDate - 6, endDate + 1) :
+      this.getUserData(id).slice(0, endDate + 1);
+  }
+
+  getMilesByDay(id, date, userRepo) {
+    let feetWalked = this.getDay(id, date).numSteps * userRepo.getUserData(id).strideLength;
+    return +(feetWalked / 5280).toFixed(1);
```

Comment:
Does the plus here just make sure its a number since .toFixed turns it into a string?

Reply:
Exactly, it's a form of type coercion